lang/java/tools/src/test/java/org/apache/avro/tool/TestConcatTool.java

```diff
@@ -136,6 +137,38 @@ public void testConcat() throws Exception {
   }
 
   @Test
+  @Ignore // AVRO-1876 is in JIRA to improve this test to use MiniDFSCluster, the following describes current state
```

Comment:
Can you convert these comment lines to a javadoc comment block on the method?

Then the Ignore annotation would look like

```
@Ignore("AVRO-1876 see method comments for details.")
```